epfast-annotations/src/test/java/com/exactpro/epfast/converter/ConverterTest.java

```diff
@@ -555,7 +554,6 @@ void stringToByteVector() {
                 .containsExactly(0x3F, 0x5F, 0x7F),
             () -> assertThat(converter.stringToByteVector("\t\r\n 3f\t5f\r7 F \t\r\n"))
                 .containsExactly(0x3F, 0x5F, 0x7F),
-
```

Comment:
I'm not that this line needed to be removed

Reply:
was there just empty line ? or was something on that line ?